### PR TITLE
Move to jekyll

### PR DESCRIPTION
**Pull request by paramt**
Use jekyll to dynamically generate site from _data/projects.csv

### TODO
 - [x] Replace JavaScript with Sass

### CODE REVIEW

**Comment by paramt** [comment]
@RDIL Do you mind taking a look?

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
+gem 'github-pages'
```
Comment (RDIL) [inline Gemfile:2]:
```suggestion
gem 'github-pages'
gem 'tzinfo-data'
```

**Review by RDIL — CHANGES_REQUESTED** [review]

File: Gemfile.lock [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+      unicode-display_width (~> 1.1, >= 1.1.1)
+    thread_safe (0.3.6)
+    typhoeus (1.3.1)
+      ethon (>= 0.9.0)
+    tzinfo (1.2.5)
+      thread_safe (~> 0.1)
+    unicode-display_width (1.6.0)
+
+PLATFORMS
+  x64-mingw32
```
Comment (RDIL) [inline Gemfile.lock:244]:
```suggestion
  x64-mingw32
  ruby
```
github pages needs this

File: Gemfile.lock [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+      ruby_dep (~> 1.2)
+    mercenary (0.3.6)
+    mini_portile2 (2.4.0)
+    minima (2.5.0)
+      jekyll (~> 3.5)
+      jekyll-feed (~> 0.9)
+      jekyll-seo-tag (~> 2.1)
+    minitest (5.11.3)
+    multipart-post (2.1.1)
+    nokogiri (1.10.3-x64-mingw32)
```
Comment (RDIL) [inline Gemfile.lock:210]:
```suggestion
    nokogiri (1.10.3)
    nokogiri (1.10.3-x64-mingw32)
```

File: Gemfile.lock [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+    em-websocket (0.5.1)
+      eventmachine (>= 0.12.9)
+      http_parser.rb (~> 0.6.0)
+    ethon (0.12.0)
+      ffi (>= 1.3.0)
+    eventmachine (1.2.7-x64-mingw32)
+    execjs (2.7.0)
+    faraday (0.15.4)
+      multipart-post (>= 1.2, < 3)
+    ffi (1.11.1-x64-mingw32)
```
Comment (RDIL) [inline Gemfile.lock:31]:
```suggestion
    ffi (1.11.1)
    ffi (1.11.1-x64-mingw32)
```

**Review by RDIL — CHANGES_REQUESTED** [review]

**Pull request merged**